docs/content/release_notes/whats_new.md

```diff
@@ -4,6 +4,20 @@ IBM® block storage CSI driver 1.11.0 introduces the enhancements that are detai
 
 **General availability date:** 15 January 2023
 
+## New dynamic volume group support
+
+The new volume group support allows dynamic management of the content of the groups. This feature is used by policy-based replication, introduced IBM Spectrum Virtualize 8.5.2 release.
+As opposed to the volume group feature present within the CSI driver which stays static on the StorageClass, the new volume group feature is dynamic. 
+
+With this new volume group support, the PersistentVolumeClaim (PVC) label can be updated at any time to change the PVC from belonging from one volume group to another, once the volume groups are defined on the storage.
+
+The advantage of using volume groups is that actions, like replication, can be done simultaneously across all volumes in a volume group.
```

Comment:
I think it's important to mention that EAR is not supported otherwise

Reply:
VG is necessary for EAR? like a prerequisite?

Reply:
per our conversation, I've added notes

Reply:
updated per webex